definitions/process_inputs.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env genome-perl
```

Comment:
As of right now, these files don't go into the github repo, since they're specific to our GMS setup